docs/deploy.sh

```diff
@@ -357,6 +357,9 @@ redirects[router/reference/not-found]=router/error-handling
 redirects[router/navigating-pages]=router/basics/navigation
 redirects[router/create-pages]=router/basics/core-concepts
 
+# After changing "Privacy Shield" to "Data Privacy Framework"
+redirects[regulatory-compliance/privacy-shield]=regulatory-compliance/data-privacy-framework
```

Comment:
Based on my previous suggestion, let's remove server-side redirect, since it won't be required when redirecting to an external link.

Reply:
I was actually thinking it could redirect to the main page, which provides the regulatory interview, and then links to the DPF